src/pages/Rockets.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRockets } from '../redux/rockets/rockets';
+import RocketList from '../components/RocketList';
+
+function Rockets() {
+  const rocketsState = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, []);
```

Comment:
- [ ] it would be better if not dispatch `fetchRockets()` every time you navigate to other pages the missions work fine but for rockets every time I leave the page I lose the reserved rockets so please try to fix that